packages-content-model/roosterjs-content-model-core/lib/publicApi/model/mergeModel.ts

```diff
@@ -384,6 +386,16 @@ function applyDefaultFormat(
     });
 }
 
+function mergeBlockFormat(applyDefaultFormatOption: string, block: ContentModelBlock) {
+    if (
+        applyDefaultFormatOption == 'keepSourceEmphasisFormat' &&
+        (block.blockType != 'BlockGroup' || block.blockGroupType != 'ListItem') &&
```

Comment:
why keep background color for list item?

Reply:
For list item we do not have background color in the format interface. 
I think we never parse it, but let be double check.

Reply:
Yes for ContentModelListItemFormat, we are not extending BackgroundColorFormat.

Reply:
How about we add BackgroundColorFormat into ContentModelListItemFormat?

Reply:
Added BackgroundColor Format to ContentModelListItemFormat as suggested.